server/services/prismic-parsers.js

```diff
@@ -383,3 +393,7 @@ export function asHtml(maybeContent: any) {
 export function isEmptyDocLink(fragment: Object) {
   return fragment.link_type === 'Document' && !fragment.data;
 }
+
+function deP(text: ?string) {
+  return text && text.replace(/<\/?p( class="")?>/g, '');
```

Comment:
Where do the p tags with empty class strings come from? I'm guessing we can't use Prismic's `asText` to achieve the same thing?

Reply:
Added note to explain why we do this.